### PR TITLE
Adding the multi-account functionality

### PR DESCRIPTION
**Pull request by Debanitrkl**
## Describe your changes
This adds the feature to use multiple accounts in a single backstage instance. 
## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.

### CODE REVIEW

File: app-config.yaml [hunk 1]
```diff
@@ -47,10 +47,22 @@ integrations:
...
 proxy:
-  '/harness':
+  '/harness/prod':
     target: 'https://app.harness.io/'
     headers:
       'x-api-key': ${HARNESS_API_KEY}
+  '/harness/qa':
+    target: 'https://qa.harness.io/'
+    headers:
+      'x-api-key': ${HARNESS_API_KEY}
```
Comment (SarvJ1) [inline app-config.yaml:57]: If you could rename the Environment Variables Names, would be better to inject them later

**Comment by Debanitrkl** [comment]
@Ompragash here's the changes, it's now able to display pipelines from [multiple accounts](https://github.com/harness/backstage-plugins/pull/77/files) with different hostname in a single backstage instance.
Here's a screenshot of CI/CD plugin displaying pipelines for a [project](https://stage.harness.io/ng/#/account/wFHXHD0RRQWoO8tIZT5YVw/home/orgs/Harness/projects/RELEASEBUILDS/) in [stage.harness.io](https://stage.harness.io/)
Screenshot 2023-03-29 at 12 25 13 PM

Also, for the documentation, I don't think we should change anything because this is just for an internal use case and doesn't change anything with the the way you should set-up the plugin inside your backstage instance.

**Review by sandeepa-kv-jois — COMMENTED** [review]
It's not supporting multiple accounts within same host/env, essentially more than one x-api-key. Are we planning this effort separately?

File: plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx [hunk 2]
```diff
@@ -745,83 +744,88 @@ function ExecutionList() {
...
   }
-  
+
   async function getPipeLineByService() {
     const list = serviceId;
     const service1 = list?.split(',').map(item => item.trim()) || '';
-  
+
     const token = getSecureHarnessKey('harnessBearerToken');
     const value = token ? `Bearer ${token}` : '';
```
Comment (jenil-harness) [inline plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx:791]: `getSecureHarnessKey` will return in format `Bearer ${token}`. We are adding Bearer again.

File: plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx [hunk 3]
```diff
@@ -745,83 +744,88 @@ function ExecutionList() {
 
   function getSecureHarnessKey(key: string): string | undefined {
     try {
-      const token = JSON.parse(decodeURI(atob(localStorage.getItem(key) || '')));
+      const token = JSON.parse(
```
Comment (jenil-harness) [inline plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx:747]: Please consider using Prettier.

**Review by jenil-harness — CHANGES_REQUESTED** [review]

**Review by Ompragash — APPROVED** [review]

**Pull request merged**